projects/experimental/components/card/card.style.less

```diff
@@ -0,0 +1,37 @@
+:host {
+    display: flex;
+    background: var(--tui-base-02);
+    border-radius: 0.75rem;
+    box-sizing: border-box;
+    padding: 0.75rem;
+    gap: 0.75rem;
+    flex-direction: column;
+
+    &[data-appearance='medium'] {
```

Comment:
Maybe it's better to use size? `<div tuiCard='m'>`?

Reply:
fixed